### PR TITLE
Update cloud-controller.md

### PR DESCRIPTION
**Pull request by mattayes**
Clean up grammar and fix inconsistencies.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7316)

### CODE REVIEW

File: docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -81,35 +79,33 @@ The Route controller is responsible for configuring routes in the cloud appropri
...
 
-The implementation of the four shared controllers highlighted above, and some scaffolding along with the shared cloudprovider interface, will stay in the Kubernetes core, but implementations specific to cloud providers will
-be built outside of the core, and implement interfaces defined in the core.
+The implementation of the four shared controllers highlighted above, and some scaffolding along with the shared cloudprovider interface, will stay in the Kubernetes core. Implementations specific to cloud providers will be built outside of the core and implement interfaces defined in the core.
 
-For more information about developing plugins, see
-[Developing Cloud Controller Manager](/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
+For more information about developing plugins, see [Developing Cloud Controller Manager](/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
 
 ## Authorization
```
Comment (mattayes) [inline docs/concepts/architecture/cloud-controller.md]: This section and its subsections are a bit confusing. The required permissions mentioned in the long description (i.e., the sentences) and the lists don't always match up. I'm new to Kubernetes and don't know which are true.

File: docs/concepts/architecture/cloud-controller.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The CCM breaks away some of the functionality of Kubernetes controller manager (
  * Route controller
  * Service controller
 
-In version 1.8, the CCM currently runs the following controllers from the preceding list:
+The CCM currently runs the following controllers from the preceding list:
```
Comment (steveperry-53) [inline docs/concepts/architecture/cloud-controller.md]: Don't say "currently". This is called out in our [style guide](https://kubernetes.io/docs/home/contribute/style-guide/). The idea is that this topic will still be around a year from now, and the reader won't know what we mean by "current".

Comment (mattayes) [reply docs/concepts/architecture/cloud-controller.md]: @steveperry-53 Thanks for sharing the style guide, I didn't know it existed! I'll replace "version 1.8" with "version 1.9" instead.

Comment (mattayes) [reply docs/concepts/architecture/cloud-controller.md]: @steveperry-53 fixed.

**Pull request merged**